#! /bin/sh -e
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Ensures to write the data on the disk when renamed a file 

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0
@DPATCH@

diff -urN prelink-0.0.20060712.orig/doc/prelink.8 prelink-0.0.20060712/doc/prelink.8
--- prelink-0.0.20060712.orig/doc/prelink.8	2004-11-23 10:37:41.000000000 +0000
+++ prelink-0.0.20060712/doc/prelink.8	2006-07-17 13:53:39.000000000 +0000
@@ -74,7 +74,7 @@
 When assigning addresses to libraries, start with random address within
 architecture dependant virtual address space range.
 This can make some buffer overflow attacks slightly harder to exploit,
-because libraries are not present on the same addresses accross different
+because libraries are not present on the same addresses across different
 machines.
 Normally, assigning virtual addresses starts at the bottom of architecture
 dependant range.
@@ -176,7 +176,7 @@
 .B \-\-md5
 This is similar to
 .I \-\-verify
-option, except instead of outputing the content of the binary or library
+option, except instead of outputting the content of the binary or library
 before prelinking to standard output MD5 digest is printed.
 See
 .BR md5sum (1).
@@ -184,7 +184,7 @@
 .B \-\-sha
 This is similar to
 .I \-\-verify
-option, except instead of outputing the content of the binary or library
+option, except instead of outputting the content of the binary or library
 before prelinking to standard output SHA1 digest is printed.
 See
 .BR sha1sum (1).
@@ -218,7 +218,7 @@
 .I \-h
 options apply), or particular ELF binaries or shared libraries.
 Unlike when walking directory hierarchies, specifying a shared library
-explicitely on the command line causes it to be prelinked even if no binary
+explicitly on the command line causes it to be prelinked even if no binary
 is linked against it. Normally, only binaries are collected together with
 all libraries they depend on.
 .SH EXAMPLES
